app-arch/createrepo_c/files/createrepo_c-1.2.1-r2-optiona_doc_c.patch

```diff
@@ -0,0 +1,45 @@
+https://github.com/rpm-software-management/createrepo_c/pull/456
+
+From 2c48ec51b7927d5258e166c3d9cff0e5beba4a32 Mon Sep 17 00:00:00 2001
+From: Jaco Kroon <jaco@uls.co.za>
+Date: Tue, 16 Sep 2025 16:44:01 +0200
+Subject: [PATCH] Make building of doxygen based C API docs configurable.
```

Comment:
This patch as it stands serves no purpose. CMake does not build the docs, regardless of whether doxygen is found, unless you run `ninja doc`. It doesn't install it no matter what.

Adding USE=doc to this ebuild when it doesn't install documentation is definitely wrong. It could be made to work, but instead of a patch you'd want to conditionally run another `cmake_build` to actually build the docs, and then also update src_install to install them.

Detecting doxygen at src_configure time even when USE=doc is disabled, would be totally harmless given that it is inert.